apps/files/package.json

```diff
@@ -25,6 +25,7 @@
     "@babel/preset-env": "^7.6.0",
     "@babel/register": "^7.6.0",
     "babel-loader": "^8.0.6",
+    "copy-to-clipboard": "^3.3.1",
```

Comment:
The problem is that `vue-clipboard2` cannot be used here when "just" importing the bundle. It needs to be initialised in the Vue instance. This would mean that we'd need to pass it as part of the context to the handler function and since this is not a feature which would be used that often probably, it felt wrong to have it in there.

But you're absolutely right, having two bundles for one feature is ugly.

Reply:
In this case, it would make sense IMO to use the new bundle instead of `vue-clipboard2`. The advantage of the latter is the ready-to-use Vue directive. With the new bundle, we'd need to either use methods then or create our own directive. If you'd be fine with this change I would go ahead and replace it. I'd do this in separate PR though not to mix different parts of the app in one PR.

Reply:
Cool, fully agree. And having it in a separate PR makes more sense. 👍